### ISSUE TITLE
Problem with redux-forms

### ISSUE THREAD

**Issue by luisherranz** [i18next/react-i18next#178]
I am having problems trying to integrate it with redux-forms.

This is my code:

```javascript
const LoginTranslated = translate('bulma')(Login);

export default reduxForm({
  form: 'login',
  fields: ['email', 'password'],
  validate,
  getFormState: state => state.bulma.reduxForm,
}, mapStateToProps)(LoginTranslated);
```

and I am getting this error:
```
Warning: setState(...): Can only update a mounted or mounting component. This usually means you called setState() on an unmounted component. This is a no-op. Please check the code for the Translate[Login] component.
```

It doesn't work doing it the other way around either:
```javascript
const LoginForms = reduxForm({
  form: 'login',
  fields: ['email', 'password'],
  validate,
  getFormState: state => state.bulma.reduxForm,
}, mapStateToProps)(Login);

export default translate('bulma')(LoginForms);
```

Any ideas?

**Comment by jamuhl** [comment]
could you try react-i18next@1.6.3?!?

**Comment by luisherranz** [comment]
It's working, thanks!

My bad for not updating first 👎

**Comment by jamuhl** [comment]
oh not your fault...i just published 1.6.3 a few minutes ago to solve your issue ;)

**Comment by luisherranz** [comment]
Then you are probably the coolest person on earth

![](https://d.gr-assets.com/hostedimages/1380222758ra/259977.gif)
